bolt-sidecar/bin/sidecar.rs

```diff
@@ -28,8 +28,7 @@ async fn main() -> eyre::Result<()> {
     let signer = Signer::new(config.private_key.clone().unwrap());
 
     let state_client = StateClient::new(config.execution_api_url.clone());
-    let mut execution_state =
-        ExecutionState::new(state_client, config.limits.max_commitments_per_slot).await?;
+    let mut execution_state = ExecutionState::new(state_client, config.limits.clone()).await?;
```

Comment:
nit: if we derive `Copy` on `Limits`, we can get rid of this clone. This doesn't change much but it's a tiny bit cleaner!